senders/telegram/send.go

```diff
@@ -31,6 +33,30 @@ var characterLimits = map[messageType]int{
 	Album:   albumCaptionMaxCharacters,
 }
 
+type Chat struct {
```

Comment:
We usually try to leave explanations for public structures, interfaces, functions, and so on

Reply:
Done https://github.com/moira-alert/moira/pull/1022/commits/2aef869c138297e2cec6315309464911ba097945